components/rangeSlider/index.html

```diff
@@ -0,0 +1,150 @@
+    </header>
+    <!-- end of header -->
+
+    <!-- start of main -->
+    <main class="grid">
+      <div class="col-1">
+        <a href="/components/avatar" class="comp-name"> Avatar</a>
+        <a href="/components/badges" class="comp-name">Badge</a>
+        <a href="/components/alert" class="comp-name">Alert</a>
+        <a href="/components/button" class="comp-name">Button</a>
+        <a href="/components/cards" class="comp-name">Cards</a>
+        <a href="/components/image/" class="comp-name">Image</a>
+        <a href="/components/input/" class="comp-name">Input</a>
+        <a href="/components/text Utilities/" class="comp-name"
+          >Text Utilities</a
+        >
+        <a href="/components/list/" class="comp-name">List</a>
+        <a href="/components/navigation/" class="comp-name">Navigation</a>
+        <a href="/components/modal/" class="comp-name">Modal</a>
+        <a href="/components/snackbar/" class="comp-name">Snackbar</a>
+        <a href="/components/rating/" class="comp-name">Rating</a>
+        <a href="/components/grid/" class="comp-name">Grids</a>
+        <a href="/components/drawer/" class="comp-name">Drawer</a>
+        <a href="/components/rangeSlider/" class="comp-name">Range Slider</a>
+      </div>
+
+      <div class="col-2">
+        <h1>Range Slider</h1>
+        <p>
+          Range sliders provides you a way of taking feedback in terms of ranges
+          from the user and it can also be used for various filters or simply to
+          communicate with the user.
+        </p>
+
+        <h3>Emoji Range Slider</h3>
+        <p>
+          HalloUi presents to you an emoji slider with easy to implement classes
+          and you can choose any emojis of your choice. The custom emoji slider
+          is suitable to take reviews or track moods of your fellows. It is a
+          fun and entertaining way of taking feedback from your users. If you
+          want to use different enojis for your slider just copy the desired
+          emojis in place of the existing ones. Do take note that whatever width
+          you will give to display-wrap or any other div you wanna wrap the
+          slider in will take its parents full width. It is designed this way so
+          that you can customise the length of the sliders.
+        </p>
+
+        <div class="display-wrap">
+          <div class="input-range-div">
+            <input type="range" class="emoji-slider" />
+            <ul class="range-values flex-row">
+              <li>☹️</li>
+              <li>😊</li>
+              <li>😍</li>
+            </ul>
+          </div>
+        </div>
+
+        <!-- pre tag -->
+
+        <pre><code class="language-markup">
+<div class="display-wrap ">
+ <div class="input-range-div">
+    <input type="range" class="emoji-slider" />
+    <ul class="range-values flex-row">
+      <li>☹️</li>
+      <li>😊</li>
+      <li>😍</li>
+    </ul>
+ </div>
+</div>
```

Comment:
Indentation missing. Please format the code.

Reply:
that is done for the ui, if i put indented code in pre tag it appears in the block with all the spaces on the left that ruins the look of the whole page.